fetchGrammarKit.bat

```diff
@@ -0,0 +1,3 @@
+@Echo Off
+:: call ant tasks that downloads and extracts grammarKit
+call ant -f prepare.xml makeToolsDir grammarKit.fetch grammarKit.extract psi.fetch  psi.extract cleanup
```

Comment:
I prefer a single ant target to do wrap all of this up.  I think that it will make it easier to maintain on multiple platforms.